cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -305,7 +305,7 @@ spec:
             - name: CLOUD_PROVIDER_API_KEY
               value: "AIzaSyDXQPG_MHUEy9neR7stolq6l0ujXmjJlvk" # The GCP Pricing API requires a key.
             - name: GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
I'm a bit hesitant to change the default path here-- this seems like it would break users of older versions. Can the old path be supported as well?